### ISSUE TITLE
Worked Once then stopped working "fetching PR failed".

### ISSUE THREAD

**Issue by emb03** [ankitbko/vscode-pull-request-azdo#60]
- Extension version: 0.0.25
- VSCode Version: 1.58.2
- OS: Big Sur

Steps to Reproduce:

1. Install Extension
2.Sign in
3. Refreshed PR List and could see my PRs
4. Closed app, reopend
5. PR say 0 for all
6. Refresh PR List says "fetching pull requests failed (401)

**Comment by emb03** [comment]
`(node:15900) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.`

**Comment by ankitbko** [comment]
This should be coming from one of the dependencies. I will check. Thank you for reporting this. But please confirm if you are able to fetch PRs now.

**Comment by emb03** [comment]
> This should be coming from one of the dependencies. I will check. Thank you for reporting this. But please confirm if you are able to fetch PRs now.

Yes, I don't get that error now, but there are no open PRs to check if this is working yet.

**Comment by emb03** [comment]
![image](https://user-images.githubusercontent.com/318121/135854836-725faaa5-0c13-426e-a496-92f5dfe77644.png)
